stack/logging.rst

```diff
@@ -2,18 +2,22 @@
 Logging
 #######
 
-This page provides guidance to developers for using logging with :lmod:`lsst.log`  in the Science Pipelines code base.
-For reference documentation on the logging framework refer to the `Doxygen page on logging`_.
+This page provides guidance to developers for using logging in the Science Pipelines code base.
+In general, all logging in Python code should be done with the standard :mod:`logging` package.
+All logging from C++ code should be done using the :lmod:`lsst.log` package and reference documentation on that logging framework can be found on the `Doxygen page on logging`_.
 For an example of configuring the logging framework in pipeline tasks, see the `pipelines.lsst.io page on logging`_.
 
 Developers are encouraged to insert log messages whenever and wherever they might be useful, with appropriate component names and levels.
 
-Whether using :lmod:`lsst.log` or any other logging mechanism, timestamps recorded in logs should use Internet `RFC 3339`_ format, which is sortable and includes the timezone.  See the discussion in `DM-1203`_ for history.
+Application code that uses both Python and C++ logging should include configuration code to forward C++ log messages to the Python :mod:`logging` system.
+This is handled automatically when using, for example, the `pipetask`_ command.
```

Comment:
Move this up to line 8?  That way you have Python, C++, and Python plus C++ in the same paragraph.